redirects-list.json

```diff
@@ -6,11 +6,13 @@
   "^/doc/start/start-github(/.*)?$                                         /doc/start/github",
   "^/doc/start/start-gitlab(/.*)?$                                         /doc/start/gitlab",
 
-  "^/doc/ref/send-github-check(/.*)?$                                      /doc/ref/check$1 301",
-  "^/doc/ref/send-comment(/.*)?$                                           /doc/ref/comment$1 301",
-  "^/doc/ref/tensorboard-dev(/.*)?$                                        /doc/ref/tensorboard$1 301",
+  "^/doc/usage$                                                            /doc/config",
 
-  "^/doc/cml-with-npm(/.*)?$                                               /doc/install$1 301",
+  "^/doc/ref/send-github-check(/.*)?$                                      /doc/ref/check$1",
```

Comment:
p.s. removes `301`s here since that's the default type of redirect (well, 90% certain).

Reply:
/CC @iterative/websites are we 100% certain?

Reply:
100% certain now. I tested it (with https://cml.dev/doc/start/start-github -> /doc/start/github) :

<img width="1221" alt="image" src="https://user-images.githubusercontent.com/1477535/203719365-a75eb71e-e0ae-45fe-89ed-995c48564191.png">